package.json

```diff
@@ -50,6 +72,8 @@
   "devDependencies": {
     "bower": "^1.7.1",
     "tslint": "^3.14.0",
+    "electron-builder": "^5.22.2",
+    "electron-prebuilt": "^1.3.2",
```

Comment:
I think it's better to execute `npm uninstall --save electron-prebuilt` before running builder then `git checkout package.json` after.  Although just a workaround.